### ISSUE TITLE
py3k option not working under Python 3 interpreter?

### ISSUE THREAD

**Issue by cosven** [PyCQA/pylint#1969]
### Steps to reproduce
1. create a file named **t.py**
```python
from urlparse import urlparse
print "hello pylint"
```
2. run pylint: `pylint --py3k t.py`

### Current behavior
pylint will not show any messages.
```
No config file found, using default configuration

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```
### Expected behavior
show messages like: bad-python3-import, print-statement

### pylint --version output

```
cosven@ubuntu:/tmp$ pylint --version
No config file found, using default configuration
pylint 1.8.3, 
astroid 1.6.2
Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
[GCC 5.4.0 20160609]
```

**Comment by PCManticore** [comment]
This happens because we only enable the porting checker on Python 2, while on Python 3 is a no-op. I think we could just do it on Python 3 as well, although some syntax checks will never be triggered, since it depends on the ast parser from Python 2.

**Comment by cosven** [comment]
I think we can firstly put an note or warning in document.

**Comment by PCManticore** [comment]
Sounds good as well! If you're up to send a patch, that would be great! If not, we'll get to it.

**Comment by cosven** [comment]
Ok, I will have a try.

**Comment by PCManticore** [comment]
the `--py3k` mode will work with Python 3 in Pylint 1.9 as well as in Pylint 2.0 once these gets launched (the code is in master respectively in the 1.9 branch)

**Comment by huguesb** [comment]
> the `--py3k` mode will work with Python 3 in Pylint 1.9 as well as in Pylint 2.0 once these gets launched (the code is in master respectively in the 1.9 branch)

This appears to be incorrect. The `--py3k` doesn't even flag `print` statements when run under a python 3 interpreter. I have tried both python 3.6 and python 3.7, and almost every release from 1.9.5 to 2.4.2 with no luck...
